examples/nD_shapes.py

```diff
@@ -43,8 +45,8 @@
         corners, shape_type='rectangle', name='broadcasted'
     )
 
-    masks = layer.to_masks(mask_shape=(128, 128, 128))
-    labels = layer.to_labels(labels_shape=(128, 128, 128))
+    masks = layer.to_masks(mask_shape=(128, 128))
+    labels = layer.to_labels(labels_shape=(128, 128))
```

Comment:
I'm confused... What's happened here?

Reply:
this is actually fixing something that was inappropriate - this layer was actually 2D and so it should not have been possible to broadcast the shape into a 3D mask. note that the other layer was actually 3D and it is still possible to get out a 3D mask.